modules.json

```diff
@@ -109,21 +79,6 @@
       "filename": "htmlwidgets-1.7.2.omod"
     },
     {
-      "name": "ID Generation",
```

Comment:
This is an id autogen, our identifiers are set manually, no dependency at all with this module, not now.